### PR TITLE
Adding  kokoro

### PR DESCRIPTION
**Pull request by pradeepthyadi-gcloud**

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```
Comment (frsv) [inline build.sh:1]: I'd suggest keeping Kokoro-related files under Kokoro folder, e.g. `kokoro/scripts`

File: src/BUILD [hunk 2]
```diff
@@ -10,7 +10,7 @@
 # distributed under the License is distributed on an "AS IS" BASIS,
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
-# limitations under the License.
+# limitations under the License.
```
Comment (frsv) [inline src/BUILD:13]: nit: This change is unrelated

File: kokoro/gcp_ubuntu_docker/kokoro_build.sh [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+#  the credentials being printed in build logs.
+#  Additionally, recursive invocation with credentials as command-line
+#  parameters, will print the full command, with credentials, in the build logs.
+# set -x
+
+# Code under repo is checked out to ${KOKORO_ARTIFACTS_DIR}/github.
+# The final directory name in this path is determined by the scm name specified
+# in the job configuration.
+cd "${KOKORO_ARTIFACTS_DIR}/github/hadoop-migration-assessment-tools"
+pwd
```
Comment (frsv) [inline kokoro/gcp_ubuntu_docker/kokoro_build.sh:19]: `pwd` and `ls -l` do not seem relevant.

What's the reason of having separate .sh file to run `bazel build //dist:all`? Is it expected to be used in other places? I would suggest just inlining the command here.

File: build.sh (hunk 1 above)
Comment (frsv) [reply build.sh:1]: This file still exists.

File: kokoro/gcp_ubuntu_docker/kokoro_build.sh (hunk 3 above)
Comment (frsv) [reply kokoro/gcp_ubuntu_docker/kokoro_build.sh]: Could you elaborate on having a separate `build.sh` file, please?

**Review by frsv — APPROVED** [review]

File: kokoro/gcp_ubuntu_docker/kokoro_build.sh (hunk 3 above)
Comment (pradeepthyadi-gcloud) [reply kokoro/gcp_ubuntu_docker/kokoro_build.sh]: we can have all the build steps in this file and actual kokoro_build will be clean
